nemo_skills/prompt/config/generic/mmlu_pro.yaml

```diff
@@ -0,0 +1,20 @@
+few_shot_examples:
+  prefix: ""
+  template: |-
+
+    Question: {question}
+    Options: {options}
+    {user_end}{assistant_begin}Answer: Let's think step by step.{solution}{assistant_end}{user_begin}
+
+  suffix: ""
+  examples_type: "{examples_type}"
+
+
+user: |-
+  The following are multiple choice questions (with answers) about {subset_for_metrics}. Think step by step and then output the answer in the format of "The answer is (X)" at the end.
+
+  {examples}
+
+
+  Question: {question}
+  Options: {options}
```

Comment:
Should {options} be on the next line? Otherwise the first option starts right here, but second and following are on newlines, which introduces a little of inconsistency

Reply:
Yes, just checked with TigerLab script
```
Options: A. ST + TS is the identity map of V onto itself.\nB. TS = 0\nC. ST = 1\nD. ST - TS = 0\nE. ST = T\nF. ST = 0\nG. ST = TS\nH. ST - TS is the identity map of V onto itself.\nI. TS = T\nJ. ST = S\nAnswer:
```